src/modules/user/user-model.ts

```diff
@@ -16,19 +16,24 @@ export interface User {
   gender: Gender
 }
 
-class UserModel {
+export interface UserModelContext {
```

Comment:
Why this interface not be named as UserModel? If it is the UserModel interface...

Reply:
Because it is necessary to export to use in other files.